### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Newt-Tea**
Added Contribution Guidelines

### CODE REVIEW

**Comment by Newt-Tea** [comment]
Added Contribution Guidelines

- Created CONTRIBUTING.md to outline contribution guidelines.
- Added detailed instructions for creating branches, committing changes, and creating pull requests.
- Specified that all pull requests require at least one review from another group member before merging.
- Included information on testing, documentation, and communication practices.
- Updated README.md to reference the new CONTRIBUTING.md file.

Changes:
- CONTRIBUTING.md: Added new file with contribution guidelines.
- README.md: Updated to include a link to the new CONTRIBUTING.md file.

**Review by aambrose1 — APPROVED** [review]
Great documentation

**Pull request merged**
